services/web/server/src/simcore_service_webserver/redis.py

```diff
@@ -64,8 +63,8 @@ async def _create_client(address: str) -> aioredis.Redis:
                     f"{client=}",
                     json.dumps(attempt.retry_state.retry_object.statistics),
                 )
-            assert client  # nosec
-            return client
+        assert client  # nosec
```

Comment:
actually you should indent them inside the ``with attempt:`` , right?

Reply:
yes correct thanks! modified it and added raise exception so mypy doesnt complain about not returning statemant.